pandas/tests/test_base.py

```diff
@@ -949,6 +949,17 @@ def test_duplicated_drop_duplicates_index(self):
                 s.drop_duplicates(inplace=True)
                 tm.assert_series_equal(s, original)
 
+    def test_drop_duplicates_series_vs_dataframe(self):
+        # GH 14192
+        df = pd.DataFrame({'a': [1, 1, 1, 'one', 'one'],
+                           'b': [2, 2, np.nan, np.nan, np.nan],
+                           'c': [3, 3, np.nan, np.nan, 'three']
```

Comment:
can u also add ``int`` and ``datetime64(with NaT)`` columns to be comprehensive?